docs/openapi.yaml

```diff
@@ -179,6 +179,79 @@ paths:
         '204':
           description: Instance is up and running
 
+  /resources/{instance}/bind-app:
+    parameters:
+      - name: name
+        in: path
+        description: Instance name
+        required: true
+        schema:
+          type: string
+
+    post:
+      summary: Binds the app to the rpaas instance
+      description: This endpoint is part of Tsuru Service API.
+      operationId: BindApp
+      tags:
+        - rpaas
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/BindApp'
+      responses:
+        '201':
+          description: App successfully bound to the rpaas instance
+        '404':
+          description: rpaas instance does not exist
+        '412':
+          description: rpaas instance not ready
+
+    delete:
+      summary: Unbinds the app from the rpaas instance
+      description: This endpoint is part of Tsuru Service API.
+      operationId: UnbindApp
+      tags:
+        - rpaas
+      requestBody:
+        required: true
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/BindApp'
+      responses:
+        '200':
+          description: App successfully unbound from rpaas instance
+        '404':
+          description: rpaas instance does not exist
+
+  /resources/{instance}/bind:
+    parameters:
+      - name: name
+        in: path
+        description: Instance name
+        required: true
+        schema:
+          type: string
+
+      post:
+        summary: Bind Unit
+        description: The service endpoint binds a unit to the service instance
+        tags:
+          - rpaas
+        responses:
+          '201':
+            description: Unit successfully bound to the service instance
+
+      delete:
+        summary: Unbind Unit
+        description: The service endpoint unbinds the unit from the service instance
+        tags:
+          - rpaas
+        responses:
+          '200':
+            description: Unit successfully unbound from service instance
+
```

Comment:
We do nothing on these routes. We ever return `200 OK` just to follow the Tsuru Service API spec... I would mark those as deprecated to indicate this.